src/odemis/acq/stream/_static.py

```diff
@@ -177,7 +177,7 @@ def __init__(self, name, raw, *args, **kwargs):
         metadata = copy.copy(raw[0].metadata)
 
         # If there are 5 dims in CTZYX, eliminate CT and only take spatial dimensions
-        if raw[0].ndim > 3:
+        if raw[0].ndim >= 3:
             dims = metadata.get(model.MD_DIMS, "ZYX")
```

Comment:
If you support more than 3 dims, you need to also check more carefully the default MD_DIMS. See the standard way in other code in odemis.

Reply:
Any specific file to check for an example?